edit-post/api/components/plugin-sidebar/style.scss

```diff
@@ -18,4 +18,8 @@
   }
 }
 
+.plugin-sidebar-error {
+  padding: 16px;
+  color: red;
```

Comment:
Do we have a better color we should be using than "red"? Or maybe not red at all? What's the design basis for this color choice?